tutorials/intro/plot_40_sensor_locations.py

```diff
@@ -96,6 +96,88 @@
+# By default a sphere  with an origin in ``(0, 0, 0)`` x, y, z coordinates and
+# radius of ``0.095`` meters (9.5 cm) is used. You can use a different sphere
+# radius by passing a single value to ``sphere`` argument in any function that
+# plots channels in 2d (like :meth:`~mne.channels.DigMontage.plot` that we use
+# here, but also for example :func:`mne.viz.plot_topomap`):
+
+biosemi_montage.plot(show_names=False, sphere=0.07)
+
+###############################################################################
+# To control not only radius, but also the sphere origin, pass a
+# ``(x, y, z, radius)`` tuple to ``sphere`` argument:
+
+biosemi_montage.plot(show_names=False, sphere=(0.03, 0.02, 0.01, 0.075))
+
+###############################################################################
+# In mne-python the head center and therefore the sphere center are calculated
+# using fiducial points. Because of this the head circle represents head
+# circumference at the nasion and ear level, and not where it is commonly
+# measured in 10-20 EEG system: above nasion at T4/T8, T3/T7, Oz, Fz level.
+# Notice below that by default T7 and Oz channels are placed within the head
+# circle, not on the head outline:
+
+biosemi_montage.plot()
+
+###############################################################################
+# If you have previous EEGLAB experience you may prefer its convention to
+# represent 10-20 head circumference with the head circle. To get EEGLAB-like
+# channel layout you would have to move the sphere origin a few centimeters
+# up on the z dimension:
+
+biosemi_montage.plot(sphere=(0, 0, 0.035, 0.094))
+
+###############################################################################
+# Instead of approximating the EEGLAB-esque sphere location as above, you can
+# calculate the sphere origin from position of Oz, Fpz, T3/T7 or T4/T8
+# channels. This is easier once the montage has been applied to the data and
+# channel positions are in head space - see this example
+# (all the code below should be a separate example):
+
+n_channels = len(biosemi_montage.ch_names)
+fake_info = mne.create_info(ch_names=biosemi_montage.ch_names, sfreq=250.,
+                            ch_types=['eeg'] * n_channels)
+data = np.random.random((n_channels, 10))
+fake_raw = mne.io.RawArray(data, fake_info)
+fake_raw.set_montage(biosemi_montage)
+
+check_ch = ['Oz', 'Fpz', 'T7', 'T8']
+pos = fake_raw._get_channel_positions(picks=check_ch)
+radius = pos[-1, 0].mean()
+x, y, z = pos[0, 0], pos[-1, 1], pos[:, -1].mean()
+
+with np.printoptions(precision=5, suppress=True):
+    print(np.array([x, y, z, radius]))
+
+fig, ax = plt.subplots(ncols=2, figsize=(8, 4))
+fake_raw.plot_sensors(axes=ax[0], show=False)
+fake_raw.plot_sensors(sphere=(x, y, z, radius), axes=ax[1], show=False)
+
+fig.texts[0].remove()
+ax[0].set_title('MNE channel projection', fontweight='bold')
+ax[1].set_title('EEGLAB channel projection', fontweight='bold')
```

Comment:
Also looks like these did not render, probably because the axes are turned off

https://21995-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/intro/plot_40_sensor_locations.html#sphx-glr-auto-tutorials-intro-plot-40-sensor-locations-py

Reply:
hm, strange, when I tested it in the notebook it looked ok:
![image](https://user-images.githubusercontent.com/8452354/91997675-a75d0a00-ed3a-11ea-9a97-a9c34dd10a9c.png)

Reply:
Try in native Python instead? I would try `tight_layout`, `subplots_adjust`, or if you're brave, `constrained_layout`

Reply:
The images on circle look cropped at the top compared with what I get locally in the notebook. Maybe the titles go outside figure - IIRC notebook usually extends the rendered space in such cases, but the saved image is still cropped (meaning: not extended). I'll add `top` `gridspec_kws` in the example then.